core/src/test/scripts/test_data/study_es_0/result_report.html

```diff
@@ -36,7 +36,7 @@
   <!--[if lt IE 9]>
     <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
   <![endif]-->
-  <title>cBioPortal validation report: study in 'test_data/study_es_0/'</title>
+  <title>cBioPortal validation report: study in '/study'</title>
```

Comment:
Is this correct? And is it odd that the only test you touched was about an HTML report without warnings or errors?